extensions/standard-processors/processors/GetFile.h

```diff
@@ -114,7 +114,7 @@ class GetFile : public core::Processor, public state::response::MetricsNodeSourc
   explicit GetFile(const std::string& name, const utils::Identifier& uuid = {})
       : Processor(name, uuid),
         metrics_(std::make_shared<GetFileMetrics>()),
-        last_listing_time_(0) {
+        last_listing_time_() {
```

Comment:
I think this can be removed

Reply:
removed it in https://github.com/apache/nifi-minifi-cpp/pull/1225/commits/78f8658fde802c73a5965dd734a544acbc37dca3#diff-6cb7e1be41456997448531e124f3d2ae5e4bc94d24c528fc1a7d2f38eb9fb685R116